### ISSUE TITLE
[Serious] Status message not automatically announced

### ISSUE THREAD

**Issue by ntallman** [APTrust/dart#631]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87014 |
| **Checkpoint** | 4.1 Compatible › Status Messages (4.1.3.a) |
| **Impact** | Serious |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 04 Jobs List |
| **URL** | <https://aptrust.org/documentation-page/dart-digital-archivists-resource-tool/> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Date Reported** | 04/21/26 at 11:41 am |
| **Issue Status** | Open |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | sharyn.morrow@ablr360.com |

## Description

Status message is not automatically announced by the screen reader.
From the Jobs List screen, if a user activates a link in the name column, a Loading progress spinner appears visually but is not announced to screen reader users.

## Recommendation

Rule : 
Screen reader users MUST be informed when a web page launches an interstitial view, a progress indicator, or enters into a paused or busy state.

How To Fix: 
Fix this issue by using ONE of the following techniques:
1. Set focus to the interstitial message
2. Announce the interstitial message via an ARIA live region

Reference: 
ARIA Live regions: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions
Accessible notifications with ARIA: https://www.sarasoueidan.com/blog/accessible-notifications-with-aria-live-regions-part-1/
WCAG Understanding document: https://www.w3.org/WAI/WCAG21/Understanding/status-messages

Background: 
When the user requests an action that will take a long time to complete, such as searching for airline tickets, it is common to present an interstitial or intermediate page to users as a way to confirm that the link worked, and that the results are coming if they wait. Screen reader and other assistive technology users need that same kind of confirmation. Moving focus to the interstitial message benefits both screen reader users and screen magnification users.

## Screenshots

![04_status-message-loading-not-announcing](https://raw.githubusercontent.com/APTrust/dart/master/docs/accessibility-audit/screenshots/87014/04_status-message-loading-not-announcing.png)
